data/en/parameterexists.json

```
@@ -4,9 +4,9 @@
 	"syntax":"parameterExists(name)",
 	"returns":"boolean",
 	"related":[],
-	"description":"This function is deprecated. Use the IsDefined or StructKeyExists functions. Determines whether a parameter exists. ColdFusion does not evaluate the argument.",
+	"description":"This function is deprecated and is still available for backward compatibility. Use StructKeyExists() or IsDefined() functions instead. \r\n\r\n Determines whether a parameter exists. ColdFusion does not evaluate the argument.",
 	"params": [
-		{"name":"region","description":"Name of the cache region.","required":true,"default":"","type":"","values":[]}
+		{"name":"parameter","description":"A syntactically valid parameter name.","required":true,"default":"","type":"","values":[]}
 	],
 	"engines": {
 		"coldfusion": {"minimum_version":"", "notes":"", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-m-r/parameterexists.html", "deprecated":"6"}
```

To answer your question (https://github.com/foundeo/cfdocs/pull/1011#issuecomment-432899433): It's set in the engines struct per engine for the specific version where they deprecated it. You could add it to the engine's notes and remove it from the description:
```suggestion
		"coldfusion": {"minimum_version":"", "notes":"This function is deprecated, but is still available for backward compatibility. Use StructKeyExists() or IsDefined() functions instead.", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-m-r/parameterexists.html", "deprecated":"6"}
```